api_specifications/note-api-0.1.0.yaml

```diff
@@ -0,0 +1,256 @@
+openapi: 3.0.1
+info:
+  title: Note And Constitute Element API Documentation
+  description: This draft API specification defines the Note and Constitute element registries.
```

Comment:
split note and constituent elements into two different YAML files

Reply:
ok